test/Libraries/RevitTestServices/TestServices.dll.config

```diff
@@ -2,6 +2,6 @@
 <configuration>
   <appSettings>
     <add key="DynamoBasePath" value=""/>
-    <add key="RequestedLibraryVersion" value="Version223"/>
+    <add key="RequestedLibraryVersion2" value="hostdefault"/>
```

Comment:
Why has the key been renamed? Where is `hostdefault` defined?

Reply:
The key has been added not renamed - the old key is parsed as a LibGVersion - while the new one is usually a `System.Version` hostdefault is a special token for letting the host decide what version to load- I've noted it in the summary for the `RequestedLibraryVersion2` property.  

That said I'm not really happy with this design, let's try to find some time tomorrow to talk. I think we can come up with something simpler.